.gitignore

```diff
@@ -6,3 +6,4 @@ assets.go
 msg.pb.go
 msg.pb.js
 msg.pb.d.ts
+.vscode
```

Comment:
That's nice and I did it on my machine. I think it's good to have it here too since VSCode is pretty popular these days :)

Reply:
I find it it best to keep the files in each repository specific to that project instead of about the people editing it. This project has no reliance on or interactions with VS Code. But it all comes down to personal preference in the end.